test/components/syncTest.js

```diff
@@ -201,3 +172,151 @@ describe('Sync Panel', function () {
     })
   })
 })
+
+describe('Syncing', function () {
+  function * setupBrave (client) {
+    Brave.addCommands()
+  }
+
+  function * setupSync (client, seed) {
+    yield setupBrave(Brave.app.client)
+    yield Brave.app.client.saveSyncInitData(seed, Immutable.fromJS([0]), 0, 'data:image/png;base64,foo')
+
+    yield Brave.app.client
```

Comment:
i think you can just do `Brave.app.client.changeSetting` to set SYNC_ENABLED to true instead of opening the prefs page and hitting the switch